apps/rpc/src/modules/event/attendance-repository.ts

```diff
@@ -52,11 +53,17 @@ export interface AttendanceRepository {
     next: AttendancePoolId
   ): Promise<void>
 
-  createAttendancePool(handle: DBHandle, attendanceId: AttendanceId, data: AttendancePoolWrite): Promise<AttendancePool>
+  createAttendancePool(
+    handle: DBHandle,
+    attendanceId: AttendanceId,
+    taskId: TaskId | null,
```

Comment:
Maybe give it a more descriptive name? mergePoolsTaskId or whatever the actual task type is named - i forgot